Src/AutoFixture/FreezeOnMatchCustomization.cs

```diff
@@ -17,6 +19,8 @@ namespace Ploeh.AutoFixture
     /// </remarks>
     public class FreezeOnMatchCustomization : ICustomization
     {
+        private object request;
```

Comment:
Why not assign this field through [a new constructor](https://github.com/AutoFixture/AutoFixture/pull/695/files#r75438371)? That way we could also make it _immutable_.